File: Other/Staircase.py
def staircase(n):
	for i in range(1, n+1):
		stair = " "*(n-i)
		stair += "#"*i
		print(stair)

def easyStaircase(n):
	for i in range(n):
		print('#'*(i+1))

staircase(6)
staircase(3)
staircase(10)

easyStaircase(6)
easyStaircase(10)

File: Other/Staircase.js
function staircase(n){
	for(var i=1; i<n+1; i++){
		var level = "";
		for(var j=0; j<n-i; j++){
			level += " ";
		}
		for(var k=0; k<i; k++){
			level += "#";
		}
		console.log(level);
	}
}

staircase(5);

function easyStaircase(n){
	for(var i=0; i<n+1; i++){
		var level = "";
		for(var j=0; j<i; j++){
			level += "#";
		}
		console.log(level);
	}
}

easyStaircase(5);